### PR TITLE
Use GitHub Issue and Pull Request templates

### PR DESCRIPTION
**Pull request by pjcozzi**
For #3596.

This initializes new issues and pull requests comments with instructions (see https://github.com/blog/2111-issue-and-pull-request-templates).  I think we'll have to merge it to see how it feels.

### CODE REVIEW

**Comment by hpinkos** [comment]
I think these templates should be more like "fill in the blank" instead of instructions.  If there are important instructions (like for issues, "post in the forum if you have a question") they should be brief bullet points at the top. Then link to the contributing guide if people want more info.

I think if it's too wordy, people aren't going to read it. Especially since it says "delete all of this" at the top.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+(Delete these instructions before hitting **Submit**.)
```
Comment (mramato) [inline .github/ISSUE_TEMPLATE.md]: For both this and the PR template, why not a concise checklist with fill in the blanks and links back to the dev guides for more information?  I think a wall of text in the wrong approach and will intimidate people, I also doubt everyone will read the whole thing or delete it before submitting.

**Comment by mramato** [comment]
@hpinkos I guess great minds think alike :smirk:

**Comment by pjcozzi** [comment]
OK, will see how it feels.

**Comment by pjcozzi** [comment]
Made some updates.  I like the PR template.  See

https://github.com/AnalyticalGraphicsInc/cesium/blob/templates/.github/PULL_REQUEST_TEMPLATE.md

The issue template is better but could still use some work if someone wants to tweak it.  See

https://github.com/AnalyticalGraphicsInc/cesium/blob/templates/.github/ISSUE_TEMPLATE.md

I'm not convinced that a fill in the blank formula will work; generally, people will just do a brain-dump into the first question (go check Firefox's bugs), but what we have now could probably be streamlined a bit.  I'm also fine with just seeing how people react to this.

**Comment by pjcozzi** [comment]
After reading [this](https://medium.com/the-javascript-collection/healthy-open-source-967fa8be7951#.cyf3nrfal):

> A lot of maintainers complain that there isn’t a more heavy handed way of forcing people to read a document before they log an issue on GitHub. We have documents all over projects in the Node.js Foundation about writing good bug reports but, first and foremost, we encourage people to log something and try to avoid putting barriers in the way of that.
>
> Sure, we get bad bugs, but we have a ton of contributors who can immediately work with people who log them to educate them on better practices and treat it as an opportunity to educate. This is why we have documentation on writing good bugs, in order to educate contributors, not as a barrier to entry.

And listening to [this](http://javascriptair.com/episodes/2016-03-02/) where they also advocate for a low barrier-to-entry and then working with contributors, especially as they show commitment and promise, I suggest we:
* Changes the default PR and issue text in this PR to just be one or two sentences that point to the Contributor's Guide, or
* Just close this

**Comment by mramato** [comment]
I'm okay with just closing this and node foundation's thoughts make sense.  Thankfully issue volume isn't too out of control for us, and while I think all maintainers wish every bug report or PR was complete and well written, this probably wouldn't help achieve that.
